qdata/ir.py

```diff
@@ -389,6 +389,9 @@ def __init__(self, op):
         super().__init__(decl_type=Declarations.GATE, op=op)
 
     def declaration_str(self):
+        if not self.goplist:
+            return f"gate {self.kwargs['op']}"[:-1] + " { }"
```

Comment:
What was the bug/edge case that required this change?

Reply:
Looks like it will just force a declaration to always have empty brackets in the case where there is no actual operations to define inside?

Reply:
Oh, that makes sense